libbeat/scripts/Makefile

```diff
@@ -397,7 +397,7 @@ package:
 
 	echo "Finished packages for ${BEATNAME}"
 
-package-dashboards:
+package-dashboards: package-setup
```

Comment:
The problem with `package-setup` is that it can take a lot of time and if `package-dashboards` is called multiple times, it is not required every time. That is also the reason it is not a dependency for `make package`